### ISSUE TITLE
Unusual way to produce string in UI?

### ISSUE THREAD

**Issue by agharbeia** [SemanticMediaWiki/SemanticMediaWiki#4903]
This issue is not based on observation of the software's behaviour. Rather it pertains to the localisation facility, and it is based on examining the strings file [i18n/en.js](/SemanticMediaWiki/SemanticMediaWiki/blob/ecb708220680a785a0e1a130bb9f474c3d733475/i18n/en.json#L513) from SMW 3.2.2.

It seems that the code pertaining to the string with id "smw-property-indicator-type-info" from the file mentioned above returns an integer to indicate the type, then the localisation mechanism of handling plural forms is used to convert that to a string.

I think this is an unexpected usage of the localisation mechanism, and could lead to unexpected results and/or confusion of the localisers. But feel free to dismiss the issue if you think otherwise.

**Comment by agharbeia** [comment]
I'm refreshing my self-maintained Arabic localisation, and I have also to say that the explanations in general, and in the statistics section have been greatly improved and allow a better understanding of the system, and hopefully more engagement from the users, not to mention architects. Thanks for that.

**Comment by Nikerabbit** [comment]
Good catch. Both of these strings fail to work for languages where singular/plurals forms are incompatible with English.

```
	"smw-page-indicator-usage-count": "Estimated [https://www.semantic-mediawiki.org/wiki/Help:Property_usage_count usage count]: {{PLURAL:$2|'''$2'''}}",
	"smw-property-indicator-type-info": "{{PLURAL:$1|User|System}} defined property",
```

At very minimum it should be `{{PLURAL:$1|1=User|System}}`, but ideally separate messages. I could not find where these messages are used in the code, so maybe they are unused and can be removed.

**Comment by kghbln** [comment]
Indeed it appears that neither ["smw-property-indicator-type-info"](https://github.com/SemanticMediaWiki/SemanticMediaWiki/search?q=smw-property-indicator-type-info) nor ["page-indicator-usage-count"](https://github.com/SemanticMediaWiki/SemanticMediaWiki/search?q=smw-page-indicator-usage-count) are active system messages for usages on property pages.

new system message is [used in this spot](https://github.com/SemanticMediaWiki/SemanticMediaWiki/blob/1f4bbda9bb8f7826ffabf00159cfdc0760043ca3/src/MediaWiki)

**Comment by agharbeia** [comment]
Good to know it's not being used.
In your estimation, how man other strings could be in the catalogue that are no being used?
(Anything to get me done with the localisation step and back on to actually developing the wiki would be great)

**Comment by kghbln** [comment]
>  In your estimation, how man other strings could be in the catalogue that are no being used?

Only a few more, if at all. Sometimes these left-overs happen in the heat of the action.
